src/pages/home/home.js

```diff
@@ -0,0 +1,21 @@
+import { Navbar } from "../../components/shared/navbar";
+import { Header } from "../../components/header/header";
+import { HomePageCard } from "../../components/home-products/homeProd-card";
+import { Categories } from "../../components/categories/categories";
+import { Footer } from "../../components/footer/footer";
```

Comment:
suggestion: you can import all these components from a single file.

Reply:
Thanks for suggestion